llama_stack/templates/cerebras/cerebras.py

```diff
@@ -8,10 +8,14 @@
 
 from llama_models.sku_list import all_registered_models
 
+from llama_stack.apis.models.models import ModelType
+
 from llama_stack.distribution.datatypes import ModelInput, Provider, ShieldInput
+from llama_stack.providers.inline.inference.sentence_transformers import (
+    SentenceTransformersInferenceConfig,
+)
```

Comment:
is the idea to use sentence transformer initially and then work with providers to see which embedding models they have?

Reply:
This is mainly for back compat reasons since we will break user flows otherwise.